### DISCUSSION TITLE
Ldproxy https Setting

### DISCUSSION THREAD

**Discussion by CHENMINGSIAN** [ldproxy/ldproxy#787] [Setup] [answered]
Ldproxy https Setting
 
Hi there,
 
I was trying to set up the HTTPS setting on the ldproxy but get the error which you can find on the attachment and wonder if you have any reference or example for this, thank you.

![image](https://user-images.githubusercontent.com/37889789/203734005-17d18542-0d98-4639-a269-c325ff451ac3.png)

![image](https://user-images.githubusercontent.com/37889789/203734153-cfb34bf1-4acb-428a-9c6a-d19414db50e1.png)

**Comment by azahnen** [comment, marked as answer]
Hi, 
this is not officially supported, nevertheless I would have expected it to work. I merged a fix, so it should work now with the recent `master` or `iide/ldproxy:next` image.

The usual way to support HTTPS is by using a reverse proxy like Apache or Traefik.

**Comment by CHENMINGSIAN** [comment]
Hi azahnen,
Many thanks for helping me set up the HTTPS  configuration. Then, there is still another issue, how can all the links which you can find below can be configured with HTTPS?

![Home](https://user-images.githubusercontent.com/37889789/204221154-a0d50cf5-2e86-4911-817e-19e6fcc79177.png)
![① 192 168 9 212restservicestest](https://user-images.githubusercontent.com/37889789/204221175-6cb964a3-d1bd-4182-983d-632af7133903.png)

**Comment by azahnen** [comment]
Hi, you have to configure an [external URL](https://docs.ldproxy.net/application/60-server.html#external-url) just as if you were actually using a reverse proxy. 
So for your example it should be `externalUrl: https://192.168.9.212/rest/services`.
